contracts/root/RootChain.sol

```diff
@@ -103,4 +130,17 @@ contract RootChain is RootChainStorage, IRootChain {
     headerBlock.createdAt = now;
     headerBlock.proposer = msg.sender;
   }
+  //   emit ChildChainChanged(childChainContract, newChildChain);
+
+  function addProofValidator(address _validator) public /* onlyOwner */  {
```

Comment:
Let's move this part to registry if it's being accessed at more than one place.

Reply:
yeah, cool did that.